### PR TITLE
Minor Power Model Doc Update

### PR DESCRIPTION
**Pull request by eslickj**
## Changes proposed in this PR:
- Title says it all.  Mostly turbine models haven't kept up with recent changes.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by MAZamarripa — APPROVED** [review]
lgtm

File: docs/technical_specs/model_libraries/power_generation/unit_models/turbine_outlet.rst [hunk 1]
```diff
@@ -121,9 +121,11 @@ Initialization
 The initialization method for this model will save the current state of the model
 before commencing initialization and reloads it afterwards.  The state of the model
 will be the same after initialization, only the initial guesses for
-unfixed variables will be changed.  To initialize this model, provide a starting
-value for the inlet port variables.  Then provide a guess for one of: discharge
-pressure, ``deltaP``, or ``ratioP``.
+unfixed variables will be changed expect for optional calculation of the flow coefficient.
```
Comment (blnicho) [inline docs/technical_specs/model_libraries/power_generation/unit_models/turbine_outlet.rst:124]: Should "expect" be "except" here?

Comment (eslickj) [reply docs/technical_specs/model_libraries/power_generation/unit_models/turbine_outlet.rst]: Yes.  Thanks!

**Review by blnicho — APPROVED** [review]

**Review by MAZamarripa — APPROVED** [review]
lgtm

**Pull request merged**